python_ta/checkers/invalid_name_checker.py

```diff
@@ -356,7 +356,7 @@ def visit_assignname(self, node: nodes.AssignName) -> None:
         if _ignore_name(node.name, self.linter.config.ignore_names):
             return
 
-        frame = node.frame(future=True)
+        frame = node.frame()
```

Comment:
For some reason the same reference on line 342 of this file to the `future` argument of `node.frame()` does not cause the same deprecation warning.